entity-framework/core/what-is-new/ef-core-8.0/breaking-changes.md

```diff
@@ -409,3 +410,26 @@ The old behavior was a bug and prevented properties that match the synthesized k
 #### Mitigations
 
 [Exclude the property from the model](xref:core/modeling/entity-properties#included-and-excluded-properties) if its value should not be persisted.
+Additionally, you can disable this behavior entirely by setting `Microsoft.EntityFrameworkCore.Issue31664` AppContext switch to `true`, see [AppContext for library consumers](/dotnet/api/system.appcontext#ForConsumers) for more details.
```

Comment:
Maybe just have a one-liner setting the appcontext swithc, which people can copy-paste directly into their project.